### PR TITLE
implemented default_agreement edit

### PR DESCRIPTION
**Pull request by deileo-datadog**

### CODE REVIEW

File: src/AppBundle/Resources/views/blocks/layout/default_agreement.html.twig [hunk 1]
```diff
@@ -118,28 +135,40 @@
...
-        {% else %}
-            <td style="width: 150px; padding-left: 5px;">Iki &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;m. &nbsp;&nbsp;&nbsp;&nbsp;mėn. &nbsp;&nbsp;&nbsp;&nbsp;d.</td>
-        {% endif %}
-        <td style="padding-left: 5px;">Už varžybų trasos licenciją:</td>
-        <td style="padding-left: 5px;">&euro; 00</td>
-    </tr>
+        <tr style="padding-left: 10px;">
+            <td style="width: 150px; padding-left: 5px;">
+                {% if not timeRestrictions.payTaxesTerm is empty %}
+                    Iki {{ timeRestriction }}
```
Comment (AivarasZ) [inline src/AppBundle/Resources/views/blocks/layout/default_agreement.html.twig]: Transale "Iki"

File: src/AppBundle/Form/ApplicationAgreementModifyType.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+namespace AppBundle\Form;
+
+use AppBundle\Entity\CmsBlock;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ApplicationAgreementModifyType extends AbstractType
+{
+
```
Comment (AivarasZ) [inline src/AppBundle/Form/ApplicationAgreementModifyType.php]: Remove empty line

File: src/AppBundle/Controller/CmsBlockController.php [hunk 3]
```diff
@@ -120,6 +122,47 @@ public function deleteAction(CmsBlock $block)
...
+     * @template("AppBundle:CmsBlock:application_agreement_modify.html.twig")
+     * @security("has_role('ROLE_ADMIN')")
+     * @param CmsBlock $block
+     * @param Request $request
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function modifyAgreementAction(CmsBlock $block, Request $request)
+    {
+        $form = $this->createForm(new ApplicationAgreementModifyType(), $block);
+
```
Comment (AivarasZ) [inline src/AppBundle/Controller/CmsBlockController.php]: Remove empty line

File: src/AppBundle/Resources/views/CmsBlock/application_agreement_modify.html.twig [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{% extends 'AppBundle::layout.html.twig' %}
+{% form_theme form 'AppBundle::forms_vertical.html.twig' %}
+
+{% block content %}
+    {% if is_granted('agreement-create', app.user) %}
+        <div class="page-header">
+            <h1>{{ 'application_agreement.new.title'|trans }}
+
```
Comment (AivarasZ) [inline src/AppBundle/Resources/views/CmsBlock/application_agreement_modify.html.twig]: Remove empty line

File: src/AppBundle/Resources/views/CmsBlock/index.html.twig [hunk 5]
```diff
@@ -41,12 +41,19 @@
             <td data-title="{{ 'cms_block.label.name'|trans }}">{{ block.name }}</td>
             <td data-title="{{ 'cms_block.label.last_update'|trans }}">{{ block.createdAt|localizeddate('short') }}</td>
             <td data-title="" class="col-min">
-              <a href="{{ url('app_cmsblock_edit', {id: block.id}) }}" class="btn btn-xs btn-link" data-title="{{ 'title.edit'|trans }}" data-toggle="tooltip">
+              {% set UrlName = 'app_cmsblock_edit' %}
```
Comment (AivarasZ) [inline src/AppBundle/Resources/views/CmsBlock/index.html.twig]: UrlName to urlName

File: src/AppBundle/Resources/views/blocks/layout/default_agreement.html.twig [hunk 6]
```diff
@@ -1,3 +1,20 @@
+<div style="display: none;">
```
Comment (AivarasZ) [inline src/AppBundle/Resources/views/blocks/layout/default_agreement.html.twig]: Why display:none?
If it must be hidden use class='hide' of bootstrap.
If we don't need this anymore, just delete this part of html.

File: src/AppBundle/Resources/views/blocks/layout/default_agreement.html.twig [hunk 7]
```diff
@@ -118,28 +135,40 @@
     </tr>
     </tbody>
 </table>
+<div style="display: none;">
```
Comment (AivarasZ) [inline src/AppBundle/Resources/views/blocks/layout/default_agreement.html.twig]: Why display:none?
If it must be hidden use class='hide' of bootstrap.
If we don't need this anymore, just delete this part of html.

File: src/AppBundle/Resources/views/blocks/layout/default_agreement.html.twig (hunk 6 above)
Comment (deileo-datadog) [inline src/AppBundle/Resources/views/blocks/layout/default_agreement.html.twig]: It is used to hide twig for and if functions from user when he's editing default agreement template threw text editor

File: src/AppBundle/Controller/CmsBlockController.php [hunk 8]
```diff
@@ -120,6 +122,46 @@ public function deleteAction(CmsBlock $block)
     }
 
     /**
+     * @route("/{id}/modify-agreement", name="app_cmsblock_modify")
```
Comment (vytautas-k) [inline src/AppBundle/Controller/CmsBlockController.php]: Annotations should start with uppercase letter

**Pull request merged**